assembl/static2/workspaces/draft-js-link-plugin/src/components/__test__/__snapshots__/AddLinkForm.spec.jsx.snap

```diff
@@ -85,7 +84,6 @@ exports[`AddLinkForm component should render a form to add a link 1`] = `
             "submitError": undefined,
             "submitFailed": false,
             "submitSucceeded": false,
-            "submitting": false,
```

Comment:
put back these snapshots changes, this is due to final-form upgrade, run `yarn` on your machine